### PR TITLE
Change copy on mid page banner for polygon round

### PR DESCRIPTION
**Pull request by RamRamez**
#3565

### CODE REVIEW

File: lang/en.json [hunk 1]
```diff
@@ -41,8 +41,8 @@
 	"component.project-stats.status.tooltip": "Projects status refers to if the project is active and raising funds, deactivated, canceled or in draft phase.",
 	"component.project-stats.verification.tooltip": "Verified projects have been reviewed by our team and found to be providing a public good, with a history of impact and reputation at stake.",
 	"component.qf-section.tooltip": "This estimation is the matching funds that this project would get if the round ended now, disregarding fraud analysis.",
-	"component.qf_middle_banner.desc": "Amplify your impact on verified projects! Donate to participating verified projects on Optimism network, earn rewards, and get your donations matched with the power of quadratic funding.",
-	"component.qf_middle_banner.title": "Quadratic Funding on Optimism",
+	"component.qf_middle_banner.desc": "The Buidl on Polygon Round aims to reward creators, builders and projects that contribute to the growth and well-being of the Polygon ecosystem. Donate to participating projects on Polygon PoS and get your donations matched with the power of quadratic funding.",
```
Comment (mohammadranjbarz) [inline lang/en.json:44]: @RamRamez  It should not be **Build** instead of **Buidl** ?

Comment (RamRamez) [reply lang/en.json:44]: @laurenluz has reviewed the copy, you can check issue #3565, but I think the "Buidl" is correct, it's something like "Hodl" in cryptocurrency
image

**Review by MohammadPCh — APPROVED** [review]
LGTM, Thanks @RamRamez

**Review by MohammadPCh — CHANGES_REQUESTED** [review]
@RamRamez Please resolve conflicts

**Review by MohammadPCh — APPROVED** [review]
Thanks @RamRamez

**Pull request merged**